cartography/intel/azure/sql.py

```diff
@@ -277,267 +291,237 @@ def load_server_details(neo4j_session, credentials, subscription_id, details, up
 
 
 @timeit
-def _load_server_dns_aliases(neo4j_session, dns_aliases, update_tag):
+def _load_server_dns_aliases(
+        neo4j_session: neo4j.Session, dns_aliases: List[Dict], update_tag: int
+) -> None:
     """
     Ingest the DNS Alias details into neo4j.
     """
     ingest_dns_aliases = """
-    MERGE (alias:AzureServerDNSAlias{id: {DNSAliasId}})
+    UNWIND {dns_aliases_list} as dns_alias
+    MERGE (alias:AzureServerDNSAlias{id: dns_alias.id})
     ON CREATE SET alias.firstseen = timestamp(), alias.lastupdated = {azure_update_tag}
```

Comment:
lastupdated should not be in the create set. it should be in update set.

Reply:
Done